draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -95,8 +97,9 @@ const Tooltip = ({
   text,
   inline,
   display = "block",
-  position,
+  position = "above",
```

Comment:
We were already defaulting to `above`, but I thought I'd make this more explicit.